sql-ballerina/Module.md

```diff
@@ -143,44 +143,71 @@ string name = "Kate";
 
 sql:ParameterizedQuery query = `INSERT INTO student(age, name)
                                 values (${age}, ${name})`;
-sql:ExecutionResultret = check dbClient->execute(query);
+sql:ExecutionResult result = check dbClient->execute(query);
 //Number of rows affected by the execution of the query.
-int? count = ret.affectedRowCount;
+int? count = result.affectedRowCount;
 //The integer or string generated by the database in response to a query execution.
-string|int? generatedKey = ret.lastInsertId;
+string|int? generatedKey = result.lastInsertId;
 }
 ```
 
 #### Querying Data
 
-This sample shows three examples to demonstrate the different usages of the `query` operation to query the
+These samples show how to demonstrate the different usages of the `query` operation to query the
 database table and obtain the results. 
 
-This example demonstrates querying data from a table in a database. 
-First, a type is created to represent the returned result set. Note the mapping of the database column 
-to the returned record's property is case-insensitive (i.e., the `ID` column in the result can be mapped to the `id` 
-property in the record). Next, the `SELECT` query is executed via the `query` remote function of the client by passing that 
-result set type. Once the query is executed, each data record can be retrieved by looping the result set. The `stream` 
-returned by the select operation holds a pointer to the actual data in the database and it loads data from the table 
-only when it is accessed. This stream can be iterated only once. 
+This sample demonstrates querying data from a table in a database.
+First, a type is created to represent the returned result set. This record can be defined as an open or a closed record
+according to the requirement. If an open record is defined, the returned stream type will include both defined fields
+in the record and additional database columns fetched by the SQL query which are not defined in the record.
+Note the mapping of the database column to the returned record's property is case-insensitive if it is defined in the
+record(i.e., the `ID` column in the result can be mapped to the `id`property in the record). Additional Column names
+added to the returned record as in the SQL query. If the record is defined as a close record, only defined fields in the
+record are returned or gives an error when additional columns present in the SQL query. Next, the `SELECT` query is executed
+via the `query` remote function of the client. Once the query is executed, each data record can be retrieved by looping 
+the result set. The `stream`returned by the select operation holds a pointer to the actual data in the database and it 
```

Comment:
```suggestion
the result set. The `stream` returned by the select operation holds a pointer to the actual data in the database and it 
```

Reply:
corrected in https://github.com/ballerina-platform/module-ballerina-sql/pull/271/commits/4b7a5e42e12db6ff25d5d8ff66db0684213fb0b3